src/components/blocks/search-form/_search-form.scss

```diff
@@ -192,35 +236,36 @@
 
     .b-search-form--etc & {
       color: base.sitecolors-siteColor("vam-black");
+      float: none;
       margin-top: 0;
+      border-radius: 0 2px 2px 0;
 
       @include mixins.breakpoints-bpMinMedium {
         @include base.typography-typeSetting(4, "regular");
       }
+
+      &::after {
+        display: none;
+      }
     }
 
-    .b-search-form--etc-gateway & {
-      border-radius: 0 2px 2px 0;
-      height: 100%;
+    .b-search-form--etc-gateway &,
+    .b-search-form--etc-search & {
```

Comment:
These 2 selectors can be reduced to one, as `.b-search-form--etc & {`

Reply:
There is already a selector in the form of `.b-search-form--etc & {` so these properties can now be combined into that.

Reply:
Change updated.